cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerModificationService.java

```diff
@@ -128,13 +130,17 @@ public void initApiClient() throws ClusterClientInitException {
     @Override
     public List<String> upscaleCluster(HostGroup hostGroup, Collection<InstanceMetaData> instanceMetaDatas)
             throws CloudbreakException {
+        LOGGER.debug("Invoking CM requests for upscale");
```

Comment:
if we add logging here, maybe we should print the parameters

Reply:
makes sense.